### PR TITLE
Add "Switch UMD" menu point to Linux version

### PR DESCRIPTION
**Pull request by TotalCaesar659**

### CODE REVIEW

File: Qt/mainwindow.cpp [hunk 1]
```diff
@@ -141,7 +142,7 @@ void MainWindow::closeAct()
 void MainWindow::openmsAct()
 {
 	QString confighome = getenv("XDG_CONFIG_HOME");
-	QString memorystick = confighome + "/ppsspp/PSP";
+	QString memorystick = confighome + "/ppsspp";
```
Comment (hrydgard) [inline Qt/mainwindow.cpp:145]: What's the motivation for this change?

Comment (TotalCaesar659) [reply Qt/mainwindow.cpp:145]: Just unification with behavior of Windows version. Besides I've found there's more folders in memstick than just PSP.
https://doodypsp.blogspot.com/2007/10/psps-directory-structure-explained_06.html

Comment (hrydgard) [reply Qt/mainwindow.cpp:145]: Oh right, forgot that's how we define it there. Good one.
